benchmark_helpers/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
My idea was to create an own repo "benchmark_helpers" where we place that stuff. But for now it just left it in the voxblox repo until it is proven to work. Wdyt?